tools/cvm-image-rewriter/plugins/02-motd-welcome/README.md

```diff
@@ -0,0 +1,3 @@
+# Motd welcome
+
+This plugin is used to set the information in `/etc/motd`, which will be displayed after successful login.
```

Comment:
If the user wants to customize the welcome information, they will need to update the content of "files/etc/motd".